src/System.ComponentModel.Composition.Registration/ref/System.ComponentModel.Composition.Registration.csproj

```diff
@@ -1,5 +1,9 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
+    <!-- this assembly is inbox in desktop, do not version it unless you
```

Comment:
Technically since we're building for `netstandard2.1` which doesn't apply to desktop we can version it.  Did you hit an error that forced you to freeze this?  I see https://github.com/dotnet/corefx/blob/80adc8b87163dfe77d5b4262ea7970cc2de87f46/src/System.ComponentModel.Composition.Registration/pkg/System.ComponentModel.Composition.Registration.pkgproj#L5, we could just remove the `net45` there.